docs/get-started/going-live.md

```diff
@@ -49,17 +49,18 @@ However, if you want to use a different database, for example, you can create a
 From the root of your project, run the following command:
 
 ```
-webiny deploy-api --env=dev
+webiny deploy api --env=dev
 ```
 
 ## 3. Deploy apps using `dev` environment
 
 From the root of your project, run the following command:
 
 ```
-webiny deploy-apps --env=dev
+webiny deploy apps --env=dev
 ```
 
+>Note: if you rename the `apps` folder to something else like `src` you can simply use `webiny deploy src --env=dev`
```

Comment:
![image](https://user-images.githubusercontent.com/5121148/81954329-0589a400-9609-11ea-9e28-786e65662f80.png)

1. I would reword the note below this snippet, and rather point out that the `apps` actually references the folder in which the deployment configuration (`resources.js`) is located. Which means that, if needed, user can create his own additional folders if needed, for example `api-public`, `apps-intranet`, etc.

2. Small error with font.